cabby/data/extract.py

```diff
@@ -101,3 +100,31 @@ def get_data_by_region(region: Text) -> Sequence:
 
     return get_wikigeo_data(wikidata_items)
 
+def split_dataset(dataset, precentage_train:float, precentage_dev:float):
```

Comment:
Wait -- this code was in the other PR I reviewed. Looks like it would be best to resolve that one and then merge the changes into this PR?

FWIW, you can create a branch of a branch so that you can keep working with code in a branch that isn't checked in yet -- probably would be useful here.

Reply:
Good to know